releasenotes/notes/dd_api_key_file-c54e01179bb6fcd9.yaml

```diff
@@ -0,0 +1,12 @@
+# Each section from every releasenote are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+enhancements:
+  - |
+    Add a new environment variable ``DD_API_KEY_FILE`` that allows to put the API in a file
+    rather than directly in an environment variable. This allows to put API key in a docker secret volume.
```

Comment:
Would it be possible to add which settings has precedence over the other one ?

Reply:
Done.